src/main/java/bhs/devilbotz/commands/DriveStraightToDock.java

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package bhs.devilbotz.commands;
+
+import bhs.devilbotz.Robot;
+import bhs.devilbotz.subsystems.DriveTrain;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.filter.LinearFilter;
+import edu.wpi.first.math.filter.SlewRateLimiter;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+/**
+ * This command is a PID controller that drives the robot straight to a set distance up a hill to
+ * dock with the charging port.
+ */
+public class DriveStraightToDock extends CommandBase {
```

Comment:
Please document the difference between `DriveStraightPID` and `DriveStraightToDock`?  Is the latter intended to eventually drive straight until a specified yaw angle is reached?